easypost/easypost_object.py

```diff
@@ -73,13 +74,15 @@ def convert_to_easypost_object(response, api_key, parent=None, name=None):
             cls = prefixes.get(cls_id[0 : cls_id.find("_")], EasyPostObject)
         else:
             cls = EasyPostObject
-        return cls.construct_from(response, api_key, parent, name)
+        return cls.construct_from(values=response, api_key=api_key, parent=parent, name=name)
     else:
         return response
 
 
 class EasyPostObject(object):
-    def __init__(self, easypost_id=None, api_key=None, parent=None, name=None, **params):
+    def __init__(
+        self, easypost_id: str = None, api_key: Optional[str] = None, parent: object = None, name: str = None, **params
```

Comment:
For these really long parameter lists, can we please use trailing commas? Black will auto format this so each is on its own line.

This is true of all throughout the repo.